.travis.yml

```diff
@@ -6,15 +6,29 @@ cache:
   directories:
    - $HOME/.m2
 
-# skip install
-install: true
+install:
+  # download Cloud SDK
+  - wget https://dl.google.com/dl/cloudsdk/channels/rapid/downloads/google-cloud-sdk-112.0.0-linux-x86_64.tar.gz
+  - tar -xzvf google-cloud-sdk-112.0.0-linux-x86_64.tar.gz
+  - export PATH=$PWD/google-cloud-sdk/bin:$PATH
+  - export CLOUDSDK_CORE_DISABLE_USAGE_REPORTING=true
+  # update all Cloud SDK components
+  - gcloud components update --quiet
+  # add App Eengine component to Cloud SDK
+  - gcloud components install app-engine-java --quiet
```

Comment:
can you file a bug against yourself to track or point to forum request at appveyor for this google-cloud-sdk install they might have on the image?